### ISSUE TITLE
Support multiple languages (i18n) for the text message copy

### ISSUE THREAD

**Issue by greggles** [google/exposure-notifications-verification-server#1377]
### TL;DR
It would be nice to be able to send text messages to people in their primary language. It looks like some work on this happened in #1107 but from the API docs I don't see how a consumer would use that.

### Design

**Proposal**
* The admin interface should allow for specifying the accepted languages and associated copy.
* The API should have a parameter for choosing one of those languages (and error if an unconfigured language is requested)

**Alternatives considered**

I think an API consumer could get a code and then do the localization locally and then send the message, but I believe the long-expires URL token is not returned in the API response (maybe I missed that in my read of the API).

**Resources**
n/a

**Additional information**
n/a

**Comment by sethvargo** [comment]
The changes made as part of https://github.com/google/exposure-notifications-verification-server/pull/1352 will enable this. The feature isn't language-specific, but you'll be able to define multiple templates. You could do different languages or different variations of messages on the same language.

/cc @whaught

**Comment by mikehelmick** [comment]
@sethvargo beat me to that by seconds...

**Comment by sethvargo** [comment]
Duplicate of https://github.com/google/exposure-notifications-verification-server/issues/1320

**Comment by greggles** [comment]
As usual, thanks so much for the quick response!
